docs/partials/demo.md

```diff
@@ -72,7 +72,7 @@ There are two important parts of every Auro component. The <a href="https://deve
 To protect from versioning conflicts with other instances of the component being loaded, it is recommended to use our `registerComponent(name)` method and pass in a unique name.
 
 ```js
-import './node_modules/@alaskaairux/auro-pane';
+import './node_modules/@aurodesignsystem/auro-pane';
```

Comment:
Please delete lines 68-92 from this file.

Reply:
Why do you want to remove the custom registration examples? Why not just add the small amount of code necessary to make theme work correctly?

Reply:
I have been implementing the custom registered component in all of the components during the repo maintenance. I dont't see why we don't keep these examples until we are actually creating and working on the one-page demo.

Reply:
Per our discussion, we will leave this example in the documentation for now.